tests/features/search_and_filters/search_and_filters.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+test.describe('Check filters and search', () => {
+  test.describe('Check filters', () => {
+      test.beforeEach(async ({steps: {pages}, page}) => {
+          await test.step(`I open catalog page`, async () => {
+              await page.goto('');
+              await pages.topPanel.goToCatalog();
+          });
+      });
+      test('Check filter Datasource', async ({steps: {pages}}) => {
```

Comment:
Could you please create a separate `describe` block for checking filters individually (could be named as "apply single filter") and move related tests into this `describe` block?

So that it will be easy to identify them